src/librustdoc/html/static/main.js

```diff
@@ -1340,7 +1340,13 @@
                 output = '<div class="search-failed"' + extraStyle + '>No results :(<br/>' +
                     'Try on <a href="https://duckduckgo.com/?q=' +
                     encodeURIComponent('rust ' + query.query) +
-                    '">DuckDuckGo</a>?</div>';
+                    '">DuckDuckGo</a>?<br/><br/>' +
+                    'Or maybe you want to take a look to the <a ' +
```

Comment:
"Take a look at" or "Look in" sounds more natural here (and in "or to" below).

Reply:
When doc searches are in other crates and not std, the "or maybe you want" might not apply as often. The following is a plain option that links to the full Rust Documentation page for providing descriptions and additional resources:

> [Rust Documentation](https://www.rust-lang.org/en-US/documentation.html) provides additional resources and guides, such as the [Rust Book](https://doc.rust-lang.org/book/index.html), [Rust by Example](https://doc.rust-lang.org/rust-by-example/), and the [Rust Reference](https://doc.rust-lang.org/stable/reference/).